Flake checks in CI fail

Looks like Flake added some additional checks and now CI fails for all new PR:

```
$ flake8 hyper test
hyper/ssl_compat.py:191:17: E741 ambiguous variable name 'O'
test/test_SSLContext.py:72:9: E722 do not use bare except'
```

--- a/hyper/ssl_compat.py
+++ b/hyper/ssl_compat.py
@@ -191,7 +191,7 @@ class SSLSocket(object):
                 O='organizationName',
                 OU='organizationalUnitName',
                 CN='commonName',
-            ).get(alias, alias)
+            ).get(alias, alias)  # noqa: E741
 
         def to_components(name):
             # TODO Verify that these are actually *supposed* to all be

--- a/hyper/ssl_compat.py
+++ b/hyper/ssl_compat.py
@@ -188,10 +188,10 @@ class SSLSocket(object):
                 C='countryName',
                 ST='stateOrProvinceName',
                 L='localityName',
-                O='organizationName',
+                O='organizationName',  # noqa: E741
                 OU='organizationalUnitName',
                 CN='commonName',
-            ).get(alias, alias)  # noqa: E741
+            ).get(alias, alias)
 
         def to_components(name):
             # TODO Verify that these are actually *supposed* to all be

--- a/test/test_SSLContext.py
+++ b/test/test_SSLContext.py
@@ -69,7 +69,7 @@ class TestSSLContext(object):
             succeeded = True
         except hyper.common.exceptions.MissingCertFile:
             threw_expected_exception = True
-        except:
+        except Exception:
             pass
 
         assert not succeeded

